objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -212,6 +234,9 @@ boolean createNeuralNet(Map<String, ?> neuralNetConfig ) {
             log.debug("No neural net type specified. Trying for default of '{}'", DEFAULT_NEURAL_NET);
         }
         
+        // Setting the outputDir value for the core, (null if it does not exist)
+        source.outputDir = (String) neuralNetConfig.get(NeuralNetInterface.OUTPUT_DIRECTORY_ENTRY);
```

Comment:
Minor, but I thought we were going to add the "type" to the interface as well, and use it a few lines above this.